src/status_im2/contexts/communities/actions/airdrop_addresses/view.cljs

```diff
@@ -0,0 +1,55 @@
+(ns status-im2.contexts.communities.actions.airdrop-addresses.view
+  (:require
+    [quo.components.list-items.account.view :as account-view]
+    [quo.core :as quo]
+    [react-native.core :as rn]
+    [status-im2.common.not-implemented :as not-implemented]
+    [status-im2.contexts.communities.actions.airdrop-addresses.style :as style]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn- render-item
+  [item]
+  [account-view/view
+   {:account-props item
+    :emoji         (:emoji item)}])
+
+(defn- accounts-list
+  [{:keys [accounts]}]
+  [rn/view {:style style/account-list-container}
+   (when (seq accounts)
+     [rn/flat-list
+      {:data      accounts
+       :render-fn render-item
+       :key-fn    :id}])])
+
+(defn- airdrop-addresses
+  [{:keys [accounts logo-uri community-name]}]
```

Comment:
Given that we already have the accounts data in our app db, I suggest directly subscribing to it here instead of passing it down as props. I'm unsure about the preferred approach for handling this. @J-Son89, what do you think? 

cc: @ilmotta

Reply:
In this case, the subscription should be here, @ajayesivan is correct.
If I follow the code correctly, in the `on-press` event we initialise view:

```
                      :on-press          #(rf/dispatch
                                           [:show-bottom-sheet
                                            {:content (fn [] [airdrop-addresses/view
                                                              {:accounts       accounts
                                                               :logo-uri       (get-in images
                                                                                       [:thumbnail :uri])
                                                               :community-name name}])}])
```

`accounts` is a subscription, so if anything changes in `accounts` the `on-press` event will change. 

The problem is that if you already clicked, and the view is displayed already and `accounts` in the db changes, this change won't be reflect in the UI, unless you close and re-open the sheet, making it effectively non-reactive.

So we should subscribe to accounts in this component, and no need to pass accounts on the on-press event. (things might change once we have a selection etc, but for now, that would be the correct implementation I believe).